### ISSUE TITLE
转载文章

### ISSUE THREAD

**Issue by luliy6** [fly51fly/aicoco#621]
> No 1. <a href="http://weibo.com/1402400261/Gxi5A1vDL">《Python深度学习》</a>
> 
> No 2. <a href="http://weibo.com/1402400261/GxcWo3uwt">朋友、导师的特别意义之一：当你自己快放弃的时候，还有人没有放弃你 ​...</a>
> 
> No 3. <a href="http://weibo.com/1402400261/Gxmznkj5a">【JavaScript实时可视化库】</a>
> 
> No 4. <a href="http://weibo.com/1402400261/GxckacxgC">Python专业性能优化技巧：用 lru_cache LRU缓存修饰器实现加速 ​...</a>
> 
> No 5. <a href="http://weibo.com/1402400261/GxkLXDtFn">漂亮的二项式展开几何可视化 ​</a>
> 
> No 6. <a href="http://weibo.com/1402400261/GxmJfD8mw">原来“小黄鸭调试法”</a>
> 
> No 7. <a href="http://weibo.com/1402400261/GxhYKyu6r">【电子书：数据科学导论】</a>
> 
> No 8. <a href="http://weibo.com/1402400261/GxmHciTEg">卓越的关键不在时间管理，而在注意力管理。 减弱注意力，集思广益，才能创意无限。 强化注意力，摒除...</a>
> 
> No 9. <a href="http://weibo.com/1402400261/Gxec1Ah0a">【非均衡学习：用Scikit-Learn应对非平衡数据】</a>
> 
> No 10. <a href="http://weibo.com/1402400261/Gxbo3xVvQ">组会上某些同学在记笔记…… ​</a>
> 
> No 11. <a href="http://weibo.com/1402400261/GxmUEajia">如果你读了篇论文或博客，发现很有用，最好能给作者发个简单的邮件，告诉他们。撰写长篇技术内容非常不易，...</a>
> 
> No 12. <a href="http://weibo.com/1402400261/Gxe1lo5J6">【梯度下降优化算法综述】</a>
> 
> No 13. <a href="http://weibo.com/1402400261/GxkHAptEu">【诡异的喜感：Nissan的“自归位”椅子】</a>
> 
> No 14. <a href="http://weibo.com/1402400261/GxkfLCLcr">3D打印：狮子 ​</a>
> 
> No 15. <a href="http://weibo.com/1402400261/GxkBIElvP">麦田与薰衣草 ​</a>
> 
> No 16. <a href="http://weibo.com/1402400261/GxlezslpW">【(PyTorch)用OpenCV实现(比Torchvision)更高效的图像增广】</a>
> 
> No 17. <a href="http://weibo.com/1402400261/GxcMmBibc">【关于(深度)学习医疗系统的前景】</a>
> 
> No 18. <a href="http://weibo.com/1402400261/GxhO19oKD">【Adobe AI面试问题汇总】</a>
> 
> No 19. <a href="http://weibo.com/1402400261/GxcISAlGx">【Geoffrey Hinton：深度学习——有望转变医疗行业的技术】</a>
> 
> No 20. <a href="http://weibo.com/1402400261/GxjBrru2a">“阶梯悖论”</a>
> 
> No 21. <a href="http://weibo.com/1402400261/GxbGIoFFq">【看代码学Python：包括一系列Python脚本集合，按主题分类，包含代码示例以及解释、各种用例和指向进一步扩展学习的链接】</a>
> 
> No 22. <a href="http://weibo.com/1402400261/GxkzDdFqg">【C语言机器学习(Cython + spaCy自然语言处理)】</a>
> 
> No 23. <a href="http://weibo.com/1402400261/GxmYAgcqE">【OCR文字(汉字)识别训练图像生成器】</a>
> 
> No 24. <a href="http://weibo.com/1402400261/Gxe1lo5J6">《An overview of gradient descent optimization algorithms》</a>
> 
> No 25. <a href="http://weibo.com/1402400261/Gxe9d05u7">“In this paper, we propose a novel architecture to ...”</a>
> 
> No 26. <a href="http://weibo.com/1402400261/GxhXJlF0W">【人人都能掌握机器学习】</a>
> 
> No 27. <a href="http://weibo.com/1402400261/GxlOemKfp">《2019 秋招的 AI 岗位竞争激烈吗？ - 知乎》</a>
> 
> No 28. <a href="http://weibo.com/1402400261/GxdfTthpC">【深度学习与搜索(摘自《面向搜索的深度学习》)】</a>
> 
> No 29. <a href="http://weibo.com/1402400261/Gxn90cSfG">NLP优秀学习资源汇总：CS224n: Natural Language Processing w...</a>
> 
> No 30. <a href="http://weibo.com/1402400261/GxdAvczdc">梵高 ​</a>
> 
> No 31. <a href="http://weibo.com/1402400261/GxjCJdMw6">莫名怀旧，像是小人书封面常用的硬笔画 ​...</a>
> 
> No 32. <a href="http://weibo.com/1402400261/GxbEUqjuD">【Facebook出品：安全的大规模Python代码重构工具】</a>
> 
> No 33. <a href="http://weibo.com/1402400261/Gxc9Atrqp">不要以为优秀、成功学者的标志，就是情愿让自己身心俱疲，这对博士生来说不是可持续性的；这种观点强化了学术圈的“毒性”</a>
> 
> No 34. <a href="http://weibo.com/1402400261/GxefpFLxB">晚安～ ​</a>
> 
> No 35. <a href="http://weibo.com/1402400261/GxmAUelOC">AI看不懂系列：(爽)啊～ ​</a>
> 
> No 36. <a href="http://weibo.com/1402400261/Gxmj9d0Nh">【桌面端商业统计图表解决方案：基于G2/React的数据可视化库】</a>
> 
> No 37. <a href="http://weibo.com/1402400261/GxjEIqu2Y">《Question Answering by Reasoning Across Documents with Graph Convolutional Networks》</a>
> 
> No 38. <a href="http://weibo.com/1402400261/GxdYesw0X">这什么P图软件？ [笑cry] ​</a>
> 
> No 39. <a href="http://weibo.com/1402400261/GxlOemKfp">好文回顾</a>
> 
> No 40. <a href="http://weibo.com/1402400261/GxbI2vHNg">'开源教程：从零开始写区块链' by Opensource Books GitHub: http:/...</a>
> 
> No 41. <a href="http://weibo.com/1402400261/GxhNffbXd">【强化学习灵活操控：高效、通用、低成本——不到10小时的学习，用深度强化学习在现实世界中灵活操控，只需廉价机器手和CPU机器】</a>
> 
> No 42. <a href="http://weibo.com/1402400261/GxkikAiNt">每次邂逅，都是奇迹 ​</a>
> 
> No 43. <a href="http://weibo.com/1402400261/GxbTRD4Ni">《深度：从这个指标来看，中国的人工智能仍落后西方很多》</a>
> 
> No 44. <a href="http://weibo.com/1402400261/Gxmw2yfBl">【(Python)轻量级配置框架：可从配置文件读取默认参数，尤其适合TensorFlow等机器学习简化默认实验参数配置】</a>
> 
> No 45. <a href="http://weibo.com/1402400261/GxcdOj23L">【(R)制图专用包】</a>
> 
> No 46. <a href="http://weibo.com/1402400261/GxhW9xZ8h">【用Dask Dataframes实现并行数据分析】</a>
> 
> No 47. <a href="http://weibo.com/1402400261/GxdSCCsmh">Wifi名字起得越来越“热辣”</a>
> 
> No 48. <a href="http://weibo.com/1402400261/GxbZzCRKT">听说45度显脸儿小？ ​</a>
> 
> No 49. <a href="http://weibo.com/1402400261/GxhoeydsP">《Learning Neural Templates for Text Generation》</a>
> 
> No 50. <a href="http://weibo.com/1402400261/GxhMxfM2z">【TPU深度学习定制的秘密】</a> 

 _Originally posted by @fly51fly in [#3](https://github.com/fly51fly/aicoco/issues/3#issuecomment-417890955)_
